### PR TITLE
New tooltip

### PR DESCRIPTION
**Pull request by anna-wix**

### CODE REVIEW

**Comment by nirhart** [comment]
I can't merge it when the build is failing

File: src/Tooltip/Tooltip.js [hunk 1]
```diff
@@ -116,26 +126,20 @@ class Tooltip extends WixComponent {
...
-        }
-      }
-      if (!this.state.visible && this.props.showTrigger === 'custom') {
-        if (nextProps.active) {
-          this.show();
-        }
-      }
-    }
-  }
+  // componentWillReceiveProps(nextProps) {
```
Comment (Faradey27) [inline src/Tooltip/Tooltip.js]: why we have commented code in production?

File: stories/Tooltip/Composite/CompositeStory.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {storiesOf} from '@kadira/storybook';
+import InteractiveCodeExample from '../../utils/Components/InteractiveCodeExample';
+import ExampleTooltip from './ExampleTooltip';
+import ExamplePopover from './ExamplePopover';
+// import ExamplePopoverMenu from './ExamplePopoverMenu';
```
Comment (Faradey27) [inline stories/Tooltip/Composite/CompositeStory.js]: why this lines commented?

File: stories/Tooltip/Composite/ExampleCustomPopover.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
+// import React, {Component, PropTypes} from 'react';
```
Comment (Faradey27) [inline stories/Tooltip/Composite/ExampleCustomPopover.js]: why we require this commented file?

**Comment by Faradey27** [comment]
@nirhart what about more strict requirments for commit names?
current commit description in master very unclear

New tooltip (#391)  …
* dummy composite stories

* dummy composite sections

* make tooltip example live

* css changes

* popover composite story almost done

* added size and some style fixes

* added `shouldCloseOnClickOutside` prop to Tooltip
finished(?) Popover composite story

* fixing eslint

* fixing eslint

* added param for not showing default props for tooltip

* default theme for tooltip

* added back  onClickOutside

* added back  onClickOutside

**Comment by nirhart** [comment]
@Faradey27 The commits inside the branches are not that important, they all will be squashed eventually.

**Comment by Faradey27** [comment]
@nirhart after squash it become one long a little ugly commit in master)
[http://oi63.tinypic.com/2aikcg9.jpg](url)

**Pull request merged**